File: 1-primeros-algoritmos/2-fibonacci/fibonacci.cpp
#include<iostream>

using namespace std;

int fibonacci(int n){
    if(n <= 2)
        return 1;
    return fibonacci(n-1) + fibonacci(n-2);
}

int main(){

    cout << "N = ";
    int n;
    cin >> n;

    cout << endl;
    cout << "Fibonacci de " << n << " es: " << fibonacci(n) << endl;
    
    return 0;
}

File: 1-primeros-algoritmos/2-fibonacci/fibonacci.py
def fibonacci(n):
    if n <= 2:
        return 1
    
    return fibonacci(n - 1) + fibonacci(n - 2)


if __name__ == '__main__':
    n = int(input("N = "))
    print(f"Fibonacci de {n} es: {fibonacci(n)}")